apps/search/src/pages/_app.tsx

```diff
@@ -30,12 +35,36 @@ const queryClient = new QueryClient({
     },
   },
 });
+type PalettesOverride = Record<"light" | "dark", SaleorThemeColors>;
+
+/**
+ * Temporary override of colors, to match new dashboard palette.
+ * Long term this will be replaced with Macaw UI 2.x with up to date design tokens
+ */
+const palettes: PalettesOverride = {
+  light: {
+    ...light,
+    background: {
+      default: "#fff",
+      paper: "#fff",
+    },
+  },
+  dark: {
+    ...dark,
+    background: {
+      default: "hsla(211, 42%, 14%, 1)",
+      paper: "hsla(211, 42%, 14%, 1)",
+    },
+  },
+};
 
 /**
  * That's a hack required by Macaw-UI incompatibility with React@18
+ *
+ * todo use from packages/shared
```

Comment:
```suggestion
 * TODO: use from packages/shared
```

It's a nitpick, but with such formatting y editor automatically highlights the line :)